tests/server/api/test_deployments.py

```diff
@@ -1390,6 +1390,84 @@ async def test_get_scheduled_runs_sort_order(
             str(flow_run.id) for flow_run in flow_runs[:3]
         ]
 
+    async def test_get_scheduled_flow_runs_updates_last_polled_time_and_status(
+        self,
+        client,
+        flow_runs,
+        deployments,
+    ):
+        deployment_1, deployment_2 = deployments
+
+        response1 = await client.get(f"/deployments/{deployment_1.id}")
+        assert response1.status_code == 200
+        assert response1.json()["last_polled"] is None
+        assert response1.json()["status"] == "NOT_READY"
+
+        response2 = await client.get(f"/deployments/{deployment_2.id}")
+        assert response2.status_code == 200
+        assert response2.json()["last_polled"] is None
+        assert response2.json()["status"] == "NOT_READY"
+
+        updated_response = await client.post(
+            "/deployments/get_scheduled_flow_runs",
+            json=dict(deployment_ids=[str(deployment_1.id)]),
+        )
+        assert updated_response.status_code == 200
+
+        updated_response_deployment_1 = await client.get(
+            f"/deployments/{deployment_1.id}"
+        )
+        assert updated_response_deployment_1.status_code == 200
+
+        assert (
+            updated_response_deployment_1.json()["last_polled"]
+            > pendulum.now("UTC").subtract(minutes=1).isoformat()
+        )
+
+        same_response_deployment_1 = await client.get(f"/deployments/{deployment_2.id}")
```

Comment:
`same_response_deployment_1` .... `client.get(f"/deployments/deployment_2.id}")`

I think you meant `same_response_deployment_2`?

Reply:
thanks yeah, that's more apt. I've applied that in [23130a0](https://github.com/PrefectHQ/prefect/pull/10969/commits/23130a0c575ca9eaa37beab0c472e182615331ef)